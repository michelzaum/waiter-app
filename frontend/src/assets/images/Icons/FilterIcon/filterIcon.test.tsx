import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest';
import { FilterIcon } from '.';

describe('FilterIcon', () => {
  it('Should render the FilterIcon component', () => {
    // given
    const result = render(<FilterIcon />);

    // then
    expect(result).toMatchSnapshot();
  });
});
