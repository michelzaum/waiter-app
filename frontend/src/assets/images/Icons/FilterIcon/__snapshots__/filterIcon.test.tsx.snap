// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`FilterIcon > Should render the FilterIcon component 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <svg
        fill="none"
        height="16"
        viewBox="0 0 16 16"
        width="16"
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          opacity="0.9"
        >
          <path
            d="M9.66738 12.0017V12.7513C9.66738 13.1342 9.4066 13.4676 9.03578 13.5603L7.36842 13.9772C6.8422 14.1086 6.33265 13.7104 6.33265 13.1682V9.00043L2.85987 5.52765C2.73449 5.40293 2.66446 5.23286 2.66446 5.05612V3.3314C2.66446 2.96325 2.96325 2.66446 3.3314 2.66446H12.6686C13.0368 2.66446 13.3356 2.96325 13.3356 3.3314V3.99835"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="1.5"
          />
          <path
            clip-rule="evenodd"
            d="M11.4101 10.1789L14.2566 7.33306C14.4634 7.1263 14.4634 6.79083 14.2566 6.58474L13.4169 5.74506C13.2102 5.53831 12.8747 5.53831 12.6686 5.74506L9.82276 8.59091C9.72338 8.69029 9.66736 8.82501 9.66736 8.96507V10.3343H11.0359C11.1767 10.3343 11.3107 10.2789 11.4101 10.1789Z"
            fill-rule="evenodd"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="1.5"
          />
        </g>
      </svg>
    </div>
  </body>,
  "container": <div>
    <svg
      fill="none"
      height="16"
      viewBox="0 0 16 16"
      width="16"
      xmlns="http://www.w3.org/2000/svg"
    >
      <g
        opacity="0.9"
      >
        <path
          d="M9.66738 12.0017V12.7513C9.66738 13.1342 9.4066 13.4676 9.03578 13.5603L7.36842 13.9772C6.8422 14.1086 6.33265 13.7104 6.33265 13.1682V9.00043L2.85987 5.52765C2.73449 5.40293 2.66446 5.23286 2.66446 5.05612V3.3314C2.66446 2.96325 2.96325 2.66446 3.3314 2.66446H12.6686C13.0368 2.66446 13.3356 2.96325 13.3356 3.3314V3.99835"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="1.5"
        />
        <path
          clip-rule="evenodd"
          d="M11.4101 10.1789L14.2566 7.33306C14.4634 7.1263 14.4634 6.79083 14.2566 6.58474L13.4169 5.74506C13.2102 5.53831 12.8747 5.53831 12.6686 5.74506L9.82276 8.59091C9.72338 8.69029 9.66736 8.82501 9.66736 8.96507V10.3343H11.0359C11.1767 10.3343 11.3107 10.2789 11.4101 10.1789Z"
          fill-rule="evenodd"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="1.5"
        />
      </g>
    </svg>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
